.github/workflows/publish_connectors.yml

```diff
@@ -27,6 +27,8 @@ jobs:
       - name: Publish modified connectors [On merge to master]
         id: publish-modified-connectors
         if: github.event_name == 'push'
+        env:
```

Comment:
I would prefer an explicit `OVERRIDE_VERSION_INCREMENT = true` that is only set if/when the workflow is launched manually.